Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
   "eng/test/mock_transport",
   "sdk/storage",
   "sdk/storage/azure_storage_blob",
+  "sdk/storage/storage_blob",
```

Comment:
This is not the right naming convention. You're `azure_storage_blob`.

Reply:
Hi Heath, that is correct. This is currently what is coming right out of the current routes and definitions in https://github.com/Azure/azure-rest-api-specs/blob/feature/blob-tsp/specification/storage/Microsoft.BlobStorage/tspconfig.yaml (aside from some stop-gap edits for NYI features). 
As we are on that topic though, is this what you are envisioning regarding the structure?
Current example for where generated `BlobBlobClient` lives: 
`sdk/storage/storage_blob/src/generated/clients/blob_blob_client.rs`

Desired destination:
`sdk/storage/azure_storage_blob/src/generated/clients/blob_blob_client.rs`

`storage_blob` would then be a module containing the generated client and models, and is defined and used in the `azure_storage_blob` crate? If so, I can pass this information along to Catalina and Joel that we are moving towards that directory structure- as presently this is what the results of the current latest typespec and emitter (`v0.4.0`) combination outputs: #1958 

As for the hand-edits to generated code, that was for illustrating what the desired changes to generated code should look like, and is being tracked here: https://github.com/Azure/typespec-rust/issues/183